mc-sema/cfgToLLVM/x86Instrs_MOV.cpp

```diff
@@ -564,6 +564,18 @@ static InstTransResult translate_MOV64ri(NativeModulePtr natM, BasicBlock *& blo
     return ret ;
 }
 
+template <int width>
+int GET_XAX() {
+  if (64 == width) {
+    return X86::RAX;
+  } else if (32 == width) {
+    return X86::EAX;
+  } else if (16 == width) {
+    return X86::AX;
+  } else {
+    return X86::AL;
```

Comment:
Maybe explicitly check for 8 == width() and throw on an `else`? My worry is that somehow this will get a width of 128 and have silent bugs.